bindings/python/src/datetime.cpp

```diff
@@ -55,6 +55,29 @@ struct time_duration_to_python
     }
 };
 
+struct time_point_to_python
+{
+    static PyObject* convert(lt::time_point pt)
+    {
+        using boost::chrono::system_clock;
+        using boost::chrono::duration_cast;
```

Comment:
my guess of why msvc is failing here is that you may need to include the ``boost/chrono.hpp`` header